string rotation
string direction
string ahead
float32 speed
time frameTimeStamp
float32 observationX
float32 observationY
float32 estimationX
float32 estimationY

